src/lib/impl/PublisherImpl.js

```diff
@@ -43,7 +43,6 @@ class PublisherImpl extends EventEmitter {
 
     this._tcpNoDelay =  !!options.tcpNoDelay;
 
-
     if (options.queueSize) {
```

Comment:
options.queueSize doesn't allow 0 queueSize!

Reply:
:+1:

Reply:
Just for the sake of git-commit historians, might want to mention that this fixes the queueSize=0 option (Infinite queueSize).